meson.build

```diff
@@ -380,30 +380,49 @@ luaold_interpreters = [
 ]
 
 lua_pc_name = ''
+have_elua = get_option('elua')
 
-if sys_windows or get_option('lua-interpreter') == 'lua'
-  if sys_windows and get_option('lua-interpreter') != 'lua'
-    warning('There is no "' + get_option('lua-interpreter')
-            + '" binding option for Windows, backing up to "lua"')
-  endif
+if get_option('lua-interpreter') == 'lua'
   config_h.set('ENABLE_LUA_OLD', '1')
   foreach l : luaold_interpreters
     lua = dependency(l[0], version: l[1], required:false)
     lua_pc_name = l[0]
-    if lua.found() == true
+    if lua.found()
       break
     endif
   endforeach
+  if not lua.found()
+    lua = dependency('lua', fallback : ['lua', 'lua_dep'])
+    if not lua.found()
+      error('Lua not found')
+    endif
+  endif
+  if have_elua
+    luaver_min = cc.compute_int('LUA_VERSION_NUM - 500',
```

Comment:
If I enable elua, I get an error here. 
```
meson.build:401:4: ERROR: Dependencies must be external dependencies
```

Reply:
Yep, `Elua` is disabled in this PR. I tried to enabled it using `lua-interpreter=luajit` and worked, but probably we should not build `elua` as it is disabled by default in master.

Reply:
But then how do you check if your changes are correct?

Reply:
Oh, right, try to enable `Evas`, as it is done in #332 .

Reply:
Well, it builds on Linux!